Beams/MRML/vtkMRMLRTBeamNode.cxx

```diff
@@ -82,7 +82,7 @@ vtkMRMLRTBeamNode::vtkMRMLRTBeamNode()
   this->CollimatorAngle = 0.0;
   this->CouchAngle = 0.0;
 
-  this->SAD = 1000.0;
+  this->SAD = 2000.0;

```

Comment:
With SAD = 1000 test PlmProtonDoseEngineTest fails.

```
======================================================================
FAIL: test_PlmProtonDoseEngineTest_FullTest1 (PlmProtonDoseEngineTest.PlmProtonDoseEngineTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/abc/devel/git/SlicerRT/PlmProtonDoseEngine/Testing/Python/PlmProtonDoseEngineTest.py", line 39, in test_PlmProtonDoseEngineTest_FullTest1
    self.TestSection_1_RunPlastimatchProtonDoseEngine()
  File "/home/abc/devel/git/SlicerRT/PlmProtonDoseEngine/Testing/Python/PlmProtonDoseEngineTest.py", line 169, in TestSection_1_RunPlastimatchProtonDoseEngine
    self.assertEqual(errorMessage, "")
AssertionError: 'SAD (=1000) must be positive and greater than aperture offset (1500)' != ''
- SAD (=1000) must be positive and greater than aperture offset (1500)
```

Reply:
Okay, the error says it has to be greater than another value. Even if you want to fix this by changing the SAD, it can be done in the test script. Please change the default back, I will take a look at this test later. Thanks!

Reply:
I returned previous value.